src/runtime/renderscript_dev.ll

```diff
@@ -0,0 +1,29 @@
+target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
```

Comment:
On further reflection I removed directives from .ll-file and instead added calls to setTargetTriple/setDataLayout from CodeGen_Renderscript_Dev::compile_to_src - in f8f6f27.